content/en/real_user_monitoring/mobile_and_tv_monitoring/ios/advanced_configuration.md

```diff
@@ -790,60 +793,34 @@ The iOS SDK first stores events locally and only uploads events when the [intake
 
 ### Clear all data
 
-You have the option of deleting all unsent data stored by the SDK with the `clearAllData` API.
+You have the option of deleting all unsent data stored by the SDK with the `Datadog.clearAllData()` API.
 
 ```swift
-   public static func clearAllData(in core: DatadogCoreProtocol = CoreRegistry.default) {
-       let core = core as? DatadogCore
-       core?.clearAllData()
-   }
+import DatadogCore
+
+Datadog.clearAllData()
 ```
 
 ### Stop data collection
 
-You can use the `StopInstance` API to stop a named SDK instance (or the default instance if the name is null) from collecting and uploading data further.
+You can use the `Datadog.stopInstance()` API to stop a named SDK instance (or the default instance if the name is `nil`) from collecting and uploading data further.
 
 ```swift
-   public static func stopInstance(named instanceName: String = CoreRegistry.defaultInstanceName) {
-       let core = CoreRegistry.unregisterInstance(named: instanceName) as? DatadogCore
-       core?.stop()
-   }
-```
+import DatadogCore
 
-### Set remote log threshold
-
-You can define the minimum log level (priority) at which to send events to Datadog in a logger instance. If the log's priority is below the one you set in this threshold, the log does not get sent. The default value is `debug` (send all logs).
-
```

Comment:
💬 This doesn't belong to RUM, it touches on [iOS Logs](https://docs.datadoghq.com/logs/log_collection/ios/?tab=swiftpackagemanagerspm) but the code snippet is very off.